frontend/model/state.js

```diff
@@ -158,6 +159,9 @@ const mutations = {
 // https://vuex.vuejs.org/en/getters.html
 // https://vuex.vuejs.org/en/modules.html
 const getters = {
+  currentGroupId (state) {
+    return state.currentGroupId
+  },
```

Comment:
Getter not needed because using `mapState` to access `state.currentGroupId` is just as easy as using the getter here. So this is creating an alternative for no good reason AFAIKT

Reply:
removed